### PR TITLE
Add tours for the general metagenomics tutorial

### PR DESCRIPTION
**Pull request by bebatut**
ping @shiltemann 

You can test them on any Galaxy instance with the tourbuilder extension

### CODE REVIEW

**Comment by shiltemann** [comment]
cool! :)

File: topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml [hunk 1]
```diff
@@ -0,0 +1,596 @@
...
+      The "Count.seqs" file keeps track of the number of sequences represented
+      by each unique representative across multiple samples. We will pass this
+      file to many of the following tools to be used or updated as needed.
+    position: left
+  - title: Quality Control
+    content: >-
+      The first step in any analysis should be to check and improve the quality
+      of our data.<br><br> For more information on the topic of quality control,
+      please see our training materials <a
+      href="https://galaxyproject.github.io/training-material/NGS-QC/">here</a>.
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml]: link broken, replace with either
https://galaxyproject.github.io/training-material//topics/sequence-analysis/tutorials/quality-control/tutorial.html for the tutorial, or link to the topic https://galaxyproject.github.io/training-material//topics/sequence-analysis for easy access to both slides and tutorial

File: topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-shotgun.yml [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+      the link. We want to rename them<br><br> For each dataset: <ol>
+        <li>Click on the pencil icon beside the file to "Edit Attributes"</li>
+        <li>Change the name in "Name" to get only the name of the sample</li>
+      </ol> 
+    position: left
+  - title: Extraction of taxonomic information
+    content: >-
+      As for amplicon data, we can extract taxonomic and community structure
+      information from shotgun data. Different approaches can be used: <ul>
+        <li>Same approaches as for amplicon data with identification and classification of OTUs. Such approaches imply a first step of sequence sorting to extract only the 16S and 18S sequences on which the same tools as for amplicon data. However, rRNA sequences represent a low proportion (< 1%) of the shotgun sequences so such approache is not the most statistically supported</li>
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-shotgun.yml]: `s/such approache is/such an approach is/` or `s/such approache is/such approaches are/`

File: topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml [hunk 3]
```diff
@@ -0,0 +1,596 @@
...
+        <li>"fasta" to the aligned output from "Align.seqs"</li>
+        <li>"count" parameter to "count_table" output from "Screen.seqs"</li>
+      </ul>
+    placement: left
+  - title: Align sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      To get an idea of the quality of the alignment, we can view the log output
+      from the summary step<br><br> <ul>
+        <li>How many sequences has been aligned?</li>
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml]: s/has/have/

File: topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml [hunk 4]
```diff
@@ -0,0 +1,596 @@
...
+        <li>"diffs" to 2</li>
+      </ul>
+    placement: left
+  - title: Perform preliminary clustering of sequences and remove undesired sequences
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      How many unique sequences are we left with after this clustering of highly
+      similar sequences?
+    placement: left
+  - title: Sequence Alignment
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-amplicon.yml]: think this doesn't belong here? already present at L233 before the alignment step?

**Comment by shiltemann** [comment]
awesome, thanks for making these @bebatut :)

**Comment by bebatut** [comment]
@shiltemann It was a good way to test in depth the tourbuilder extension :)
Can we merge this one?

**Pull request merged**